### PR TITLE
Make Google Play badge readable, add trademark notice

### PR DESCRIPTION
**Pull request by armata**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * not applicable
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description
Changes the size of Google Play badge. Adds a trademark notice.

### CODE REVIEW

**Comment by armata** [comment]
@moxie0 can you please check and/or merge this? should be fine.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Signal uses your phone's data connection (WiFi/3G/4G) to communicate securely, o
 
 Currently available on the Play store.
 
-<a href="https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"><img alt="Get it on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" height=36px /></a>
+<a href='https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms&hl=en&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png' width='180px'/></a>
```
Comment (mueller-ma) [inline README.md]: Why not just "https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms" ?

Comment (armata) [inline README.md]: The URL was automatically generated using the [Google Play badge](https://play.google.com/intl/en_us/badges/) form. (though this time 'hl=en' was missing...?)

Comment (mueller-ma) [inline README.md]: I would remove hl=en. Why should EVERYONE see this page in en?

Comment (armata) [inline README.md]: I've removed it.

**Comment by armata** [comment]
@moxie0 this can be merged

**Comment by armata** [comment]
PTAL: @moxie0

**Pull request closed without merging**